azurerm/resource_arm_dns_zone.go

```diff
@@ -25,12 +25,7 @@ func resourceArmDnsZone() *schema.Resource {
 				ForceNew: true,
 			},
 
-			"resource_group_name": {
-				Type:             schema.TypeString,
-				Required:         true,
-				ForceNew:         true,
-				DiffSuppressFunc: resourceAzurermResourceGroupNameDiffSuppress,
-			},
+			"resource_group_name": resourceGroupNameSchema(),
```

Comment:
shouldn't this be `resourceGroupNameDiffSupressSchema()`?

Reply:
Good find! Changing it